inference-engine/src/transformations/src/transformations/smart_reshape/proposal_scales_stridedslice.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <transformations/smart_reshape/proposal_scales_stridedslice.hpp>
+
+#include <ngraph/ngraph.hpp>
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/pattern/matcher.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ProposalScales, "ProposalScales", 0);
+
+ngraph::pass::ProposalScales::ProposalScales() {
+    auto parameter_label = ngraph::pattern::wrap_type<opset5::Parameter>({}, [](const Output<Node> &output) {
```

Comment:
You can skip first argument
```suggestion
    auto parameter_label = ngraph::pattern::wrap_type<opset5::Parameter>([](const Output<Node> &output) {
```